### PR TITLE
Update build.yml (hopefully slightly simplified)

### PR DESCRIPTION
**Pull request by blubbsy**
Here agian the text from https://github.com/jakkra/ZSWatch-HW/pull/73:

I "updated" the kibot action to generate the production files:

changed to use the latest ubuntu version
changed to use the latest kibot thingie.
Reduced the dependencies to only check out Kampi's library
--> the oficial kibot container, has all dependencies included

i dont know why, but the artificat zip files are bigger now, but i unpacked them and couldnt find any difference (the size is then identical). I assume that it has something to do with the container...

Does it make anything better? To be honest: i dont think so, but i hope that using the container reduces potential compatibility problems or to find potential problems with an KiCAD update.

### CODE REVIEW

**Comment by blubbsy** [comment]
my git skills have to seriously improve... still i dont get why it wants to commit again the library path or thinks that it differs, but okay.

**Review by jakkra — COMMENTED** [review]
Over than that I'll let @Kampi review. But from my side it looks like a great "Cleanup", thanks!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,7 +8,8 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```
[the remark this answers is not in the document]
Comment (Kampi) [reply .github/workflows/build.yml:11]: Please don't use latest. Let's stick to one version, test everything with it and then migrate to a newer version later.

**Comment by Kampi** [comment]
Sorry I missed it completely because I removed it from my task list accidentally. I will check it later on my PC.

File: .github/workflows/build.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (blubbsy) [reply .github/workflows/build.yml:11]: ok. does it make sense to switch to Ubuntu 24.04 as it is the newest LTS version. Apparently with latest it is working at the moment, so there is no need to switch again in the next 5 years.

[the remark this answers is not in the document]
Comment (Kampi) [reply .github/workflows/build.yml:11]: Imo we can use the latest LTS if it's working.

**Pull request merged**